mathics/core/convert/op.py

```diff
@@ -27,6 +29,57 @@
 operator_to_ascii = OPERATOR_CONVERSION_TABLES["operator-to-ascii"]
 
 
+amstex_operators = {
+    "\u2032": "'",
+    "\u2032\u2032": "''",
+    "\u2062": " ",
+    "\u221e": r"\infty ",
+    "\u00d7": r"\times ",
+    "(": r"\left(",
+    "[": r"\left[",
+    "{": r"\left\{",
+    ")": r"\right)",
+    "]": r"\right]",
+    "}": r"\right\}",
+    "\u301a": r"\left[\left[",
+    "\u301b": r"\right]\right]",
+    ",": ",",
+    ", ": ", ",
+    "\u222b": r"\int",
+    "\u2146": r"\, d",
+    "\uF74C": r"\, d",
+    "\U0001D451": r"\, d",
+    "\u2211": r"\sum",
+    "\u220f": r"\prod",
+    "\u03b1": r"\alpha",
```

Comment:
Here I included all the greek letters, because I could not find the tables where the non-operator Unicode characters are translated to amslatex.

Reply:
See https://github.com/Mathics3/mathics-scanner/blob/master/mathics_scanner/data/named-characters.yml#L260-L261

If we need to put in JSON another table, let's do that.

Reply:
They are in character tables, but they were not loaded as dictionaries. I leave to see how to load them the other parts work.

Reply:
https://github.com/Mathics3/mathics-scanner/pull/117 adds `unicode-to-amslatex` into the character-tables.json. Let's use that.